src/ethereum/info/fantomInfo.ts

```diff
@@ -295,6 +295,10 @@ const networkInfo: EthereumNetworkInfo = {
         'https://lb.drpc.org/ogrpc?network=fantom&dkey={{drpcApiKey}}'
       ],
       ethBalCheckerContract: '0x07f697424abe762bb808c109860c04ea488ff92b'
+    },
+    {
+      type: 'evmscan',
```

Comment:
The commit message says "Add FTM and ETHW evmscan servers", but this change is only for FTM?